tests/test_integration.py

```diff
@@ -160,6 +161,23 @@ def check_vm_absent(self, hv_name=None):
                     )
 
 
+class HypervisorCpuThresholdsTest(IGVMTest):
+    """Test that all hardware_models of all hypervisors have a cpu threshold
+    value defined in HYPERVISOR_CPU_THRESHOLDS dict"""
+
+    def test_hypervisor_cpu_thresholds(self):
```

Comment:
When we purchase new hardware we don't run igvm integration tests. So this does not necessarily have to be tested before we go to production.

However if we insist on suck check then it should be extended to cover all config variables where hardware or cpu model is checked, like KVM_HWMODEL_TO_CPUMODE.

Reply:
I extended the class to now have two tests. One for HYPERVISOR_CPU_THRESHOLDS and one for KVM_HWMODEL_TO_CPUMODEL

I also added a setUp function to share the list of hardware_models between the two tests.
Is it correct to do it like this?

Reply:
`setUp` is good to share code between tests, yes. just keep in mind that it will still be executed once for each test. but that doesn't matter i would say

Reply:
also good that we have integration tests for this now 👍

Reply:
Resolving this comment as it seems fixed now